sdk/openai/openai/samples-dev/batch.ts

```diff
@@ -26,9 +25,11 @@ export async function main() {
   const apiVersion = "2024-08-01-preview";
   const client = new AzureOpenAI({ azureADTokenProvider, deployment, apiVersion });
 
+  const batchContent = `{ "custom_id": "request-1", "method": "POST", "url": "/v1/chat/completions", "body": { "model": "${deployment}", "messages": [{ "role": "system", "content": "You are a helpful assistant." }, { "role": "user", "content": "What is 2+2?" }] } }`;
```

Comment:
Are there types we could use to type this definition?

Reply:
I cannot find an appropriate type for this. There is a section on the OpenAI reference page [here](https://platform.openai.com/docs/api-reference/batch/request-input), but there is no type associated with this